test/recipes/aws-parallelcluster-install/controls/users.rb

```diff
@@ -0,0 +1,22 @@
+user = "pcluster-admin"
+
+control 'admin_user' do
+  title 'Configure cluster admin user'
+
+  describe command("grep #{user} /etc/passwd") do
+    its('stdout.strip') { should eq "#{user}:x:400:400:AWS ParallelCluster Admin user:/home/#{user}:/bin/bash"}
+  end
+
+  describe user(user) do
+    its(:gid) { should eq 400 }
+    its(:group) { should eq "pcluster-admin" }
+  end
```

Comment:
I like it!

Reply:
I explicitly leave them separated as they are logically different